plugins/sentry/src/api/annotations.ts

```diff
@@ -17,7 +17,12 @@
 import { Entity } from '@backstage/catalog-model';
 
 export const SENTRY_PROJECT_SLUG_ANNOTATION = 'sentry.io/project-slug';
+export const SENTRY_ORGANIZATION_ANNOTATION = 'sentry.io/organization';
```

Comment:
Can't we bake this into the other annotation? So you can specify `my-org/my-project`, but in a backwards compatible way such that things still work if it doesn't have that first part.

Would update the docs here then https://github.com/backstage/backstage/blob/179a5906560a8d5b2cc4ff3b229dd76a91387eca/docs/features/software-catalog/well-known-annotations.md?plain=1#L249

Reply:
Done!